### ISSUE TITLE
pep8

### ISSUE THREAD

**Issue by thorade** [UdK-VPT/CoTeTo#4]
We should run the tool autopep8 from time to time:
```
cd CoTeTo
flake8 --count --max-line-length=120 --statistics -qq
autopep8 --in-place --recursive --max-line-length=120 --select="E101,W1,W2" .
autopep8 --in-place --recursive --max-line-length=120 --select="E101,E3,W1,W2,W3" .
```
and later for other rules, run something like
```
flake8 --count --max-line-length=100 --statistics -qq
flake8 --count --max-line-length=100 --select="E711"
autopep8 --in-place --recursive --max-line-length=100 --select="E231" .
autopep8 --in-place --recursive --max-line-length=100 --ignore="E402,F401,F403,F405" .
autopep8 --in-place --recursive --max-line-length=100 .
autopep8 --in-place --recursive --max-line-length=100 --aggressive .
autopep8 --in-place --recursive --max-line-length=100 --aggressive --aggressive --aggressive .
```

https://pypi.python.org/pypi/autopep8#features
https://pypi.python.org/pypi/flake8

**Comment by thorade** [comment]
Or just use the PyCharm reformat code functionality, seems to be doing the same.

**Comment by thorade** [comment]
closed by PR #6
